src/main/java/org/gelecekbilimde/scienceplatform/auth/model/request/RoleChangeRequestsFilter.java

```diff
@@ -0,0 +1,14 @@
+package org.gelecekbilimde.scienceplatform.auth.model.request;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class RoleChangeRequestsFilter {
```

Comment:
@coderabbitai  We don't generally recommend the `@Data `annotation because it includes unused annotations. You can review this article as an example: 
 https://medium.com/@menekseyuncu/gereksiz-anotasyon-kullan%C4%B1m%C4%B1n%C4%B1n-spring-boot-performans%C4%B1-%C3%BCzerindeki-etkisi-45846e08ea2a